pkg/scheduler/framework/types.go

```diff
@@ -808,6 +808,11 @@ func (n *NodeInfo) FilterOutPods(pods []*v1.Pod) []*v1.Pod {
 		return pods
 	}
 	filtered := make([]*v1.Pod, 0, len(pods))
```

Comment:
`FilterOutPods` is no longer used now, maybe we should remove them directly. @Huang-Wei

Reply:
Sorry ,I didn't notice that before.Let me know if you want to close this pr.

Reply:
can you please remove the function completely

Reply:
> can you please remove the function completely

Done